tests/language/null.liq

```diff
@@ -0,0 +1,18 @@
+#!../../src/liquidsoap ../../libs/pervasives.liq
+
+%include "test.liq"
+
+def t(x,y) =
+  if x != y then test.fail() end
+end
+
+def f() =
+  t(null() ?? "bla", "bla")
+  t(null("foo") ?? "bla", "foo")
```

Comment:
I'm a little bit worried about this being potentially confusing but let's go with it for now.